hosting/single/Dockerfile

```diff
@@ -20,6 +20,8 @@ RUN node /pinVersions.js && yarn && yarn build && /cleanup.sh
 
 FROM couchdb:3.2.1
 
+ARG TARGETARCH
```

Comment:
Can we default this to to x86-64 ? And it can be over-ridden for arm64. Also maybe add a comment as to the available architectures.

Reply:
By default this will be amd64, this is an argument that is passed in my `buildx` defining which build it is performing, I don't think we have control over it really.

For Oracle etc we have an install script which works out based on the `TARGETARCH` (it just checks if it starts with ARM, then uses the ARM binaries, otherwise it uses the x86/64 bit binaries) for MinIO we could have a similar script which does this - you are right that using the arch directly could be a little unsafe.